docs/_templates/include/2019/australia-sponsors.html

```diff
@@ -0,0 +1,52 @@
+<div class="sponsors">
```

Comment:
I don't believe we need these files anymore, we might need to update the site to match the others though.